distribution/changelog.txt

```diff
@@ -25,6 +29,7 @@
 - Fix: [#19112] Clearing the last character in the Object Selection filter does not properly reset it.
 - Fix: [#19114] [Plugin] GameActionResult does not comply to API specification.
 - Fix: [#19136] SV6 saves with experimental RCT1 paths not imported correctly.
+- Removed: [project#1] Snakes.
```

Comment:
Refactors are never put in the log. Also we didn't remove Herobrine.

Reply:
Dang it, those darn herobrines.